lib/ansible/plugins/action/net_base.py

```diff
@@ -107,18 +107,12 @@ def run(self, tmp=None, task_vars=None):
                 display.vvvv('wrong context, sending exit to device', self._play_context.remote_addr)
                 conn.send_command('exit')
 
-        if 'fail_on_missing_module' not in self._task.args:
-            self._task.args['fail_on_missing_module'] = False
```

Comment:
Could we not just change this to `True` instead? I get erroring by default, but there are clearly cases where one might want the current functionality.

Reply:
That seems like a good idea.

Reply:
Fixed

Reply:
Fixed